src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -148,25 +151,17 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
             }
         }
 
-        private ConnectionInformation GetConnectionInformation()
```

Comment:
No longer required. The calling code already has the _BoundSonarQubeProject_ so it can just call the _CreateConnectionInformation_ extension method directly.